EECS3311-PD1/src/main/java/UI/Main.java

```diff
@@ -1,19 +1,26 @@
 package UI;
 
 import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
-import javafx.scene.control.Label;
-import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
 
+import java.io.IOException;
+
 public class Main extends Application {
 
     @Override
     public void start(Stage stage) {
-        String javaVersion = System.getProperty("java.version");
-        String javafxVersion = System.getProperty("javafx.version");
-        Label l = new Label("Hello, JavaFX " + javafxVersion + ", running on Java " + javaVersion + ".");
-        Scene scene = new Scene(new StackPane(l), 640, 480);
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml/MainPane.fxml"));
+        Parent pane;
+        try {
+            pane = loader.load();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        Scene scene = new Scene(pane);
         stage.setScene(scene);
         stage.show();
     }
```

Comment:
Keep this file unchanged, we will change it later when we are integrating things together